content/desktop/release-notes.md

```diff
@@ -24,6 +24,36 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.27.2
+
+{{< release-date date="2024-02-08" >}}
+
+{{< desktop-install all=true version="4.27.2" build_path="/LO-TBD/" >}}
+
+### Upgrades
+
+- [Compose v2.24.5](https://github.com/docker/compose/releases/tag/v2.24.5)
+- [Docker Scout CLI v1.4.1](https://github.com/docker/scout-cli/releases/tag/v1.4.1)
+- Docker Debug v0.0.24
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug where the diagnostics ID would not print correctly when uploading diagnostics from the terminal.
+- Fixed a bug where the default settings values were being reset to default on startup, when using Settings Management.
+- Docker Scout:
+  - Updated dependencies to address Leaky Vessels series of CVEs ([CVE-2024-21626](https://github.com/advisories/GHSA-xr7r-f8xq-vfvv), [CVE-2024-24557](https://github.com/advisories/GHSA-xw73-rw38-6vjc))
+  - Added initial VEX document to document false positive [CVE-2020-8911](https://github.com/advisories/GHSA-f5pg-7wfw-84q9) and [CVE-2020-8912](https://github.com/advisories/GHSA-7f33-f4f5-xwgw)
+  - Added support for cosign SBOM attestations
+  - Added support for VEX in-toto attestations
+
```

Comment:
- Docker Debug:
  - Fixed pulling the image behind resource accesses management
  - Fixed connection issues

Link: https://docs.docker.com/engine/reference/commandline/debug/

cc @usha-mandya for better phrasing 😅

Reply:
Updated, PTAL!